.gitmodules

```diff
@@ -1,3 +1,6 @@
 [submodule "optiga-trust-m"]
 	path = optiga-trust-m
 	url = https://github.com/Infineon/optiga-trust-m
+[submodule "optiga-trust-m-1"]
```

Comment:
i know you did this for the timebeing, so please move to the trust-m repo, not trust-m-1